apps/src/netsim/NetSimSendPanel.js

```diff
@@ -412,6 +412,10 @@ NetSimSendPanel.prototype.onSendEventTriggered_ = function (jQueryEvent) {
   if (level.messageGranularity === MessageGranularity.PACKETS) {
     this.beginSendingPackets_();
   } else if (level.messageGranularity === MessageGranularity.BITS) {
+    if (this.getNextBit_() === undefined) {
+      // Ignore send events in BITS mode when there are no bits queued.
```

Comment:
Originally, because I wanted to emulate the `if (triggeringTarget.is('[disabled]')) { return; }` behavior above; but only in the BITS case.

I debated about this for a bit, and decided I like that the contract of `sendOneBit_()` is to send a bit, or throw if it's not possible right now.  That said, it looks like the only place we call `sendOneBit_()` is from here in `onSendEventTriggered_()` and we don't catch those errors anywhere, so maybe it's better to say "if the next bit is undefined, then sending nothing silently is a success."